controllers/utils.go

```diff
@@ -73,9 +72,11 @@ func parseJobYAML(yamlData []byte) (*batchv1.Job, error) {
 	return job, nil
 }
 
-func readJobYAML(jobFileName string) ([]byte, error) {
-	jobFilePath := filepath.Join(peerpodsImageJobsPathLocation, jobFileName)
-	yamlData, err := os.ReadFile(jobFilePath)
+// Method to read yaml file.
+// The full path of the job yaml file is passed as an argument
```

Comment:
```suggestion
// The full path of the yaml file is passed as an argument
```
since this is a generic method.